site/assets/scss/_syntax.scss

```diff
@@ -1,108 +1,123 @@
-/* LiteralNumberOct */ .chroma .mo { color: $green-700; }
-/* LiteralString */ .chroma .s { color: $red-500; }
-/* LiteralStringAffix */ .chroma .sa { color: $blue-600; }
-/* LiteralStringBacktick */ .chroma .sb { color: $red-500; }
-/* LiteralStringChar */ .chroma .sc { color: $red-500; }
-/* LiteralStringDelimiter */ .chroma .dl { color: $blue-600; }
-/* LiteralStringDoc */ .chroma .sd { font-style: italic; color: $red-500; }
-/* LiteralStringDouble */ .chroma .s2 { color: $red-500; }
-/* LiteralStringEscape */ .chroma .se { color: $red-500; }
-/* LiteralStringHeredoc */ .chroma .sh { color: $red-500; }
-/* LiteralStringInterpol */ .chroma .si { color: $red-600; }
-/* LiteralStringOther */ .chroma .sx { color: $red-500; }
-/* LiteralStringRegex */ .chroma .sr { color: $teal-700; }
-/* LiteralStringSingle */ .chroma .s1 { color: $red-500; }
-/* LiteralStringSymbol */ .chroma .ss { color: $yellow-800; }
-/* Name */ .chroma .n { color: $green-700; }
-/* NameAttribute */ .chroma .na { color: $green-700; }
-/* NameBuiltin */ .chroma .nb { color: $green-700; }
-/* NameBuiltinPseudo .chroma .bp { } */
-/* NameClass */ .chroma .nc { color: $blue-600; }
-/* NameClass */ .chroma [class*="css"] .nc { font-weight: $font-weight-bold; }
-/* NameConstant */ .chroma .no { color: $green-700; }
-/* NameDecorator */ .chroma .nd { color: $purple-700; }
-/* NameEntity */ .chroma .ni { color: $gray-800; }
-/* NameException */ .chroma .ne { color: $red-500; }
-/* NameFunction */ .chroma .nf { color: $teal-700; }
-/* NameFunctionMagic .chroma .fm { } */
-/* NameLabel */ .chroma .nl { color: $purple-700; }
-/* NameNamespace */ .chroma .nn { color: $red-500; }
-/* NameOther */ .chroma .nx { color: $purple-700; }
-/* NameProperty .chroma .py { } */
-/* NameTag */ .chroma .nt { color: $blue-600; }
-/* NameTag */ .chroma .language-html .nt,
-.chroma .language-css .nt { font-weight: $font-weight-bold; }
-/* NameVariable */ .chroma .nv { color: $red-500; }
-/* NameVariableClass .chroma .vc { } */
-/* NameVariableGlobal .chroma .vg { } */
-/* NameVariableInstance .chroma .vi { } */
-/* NameVariableMagic .chroma .vm { } */
-/* Operator */ .chroma .o { color: $gray-700; }
-/* OperatorWord */ .chroma .ow { color: $black; }
-/* Punctuation */ .chroma .p { color: $gray-700; }
-/* TextWhitespace */ .chroma .w { color: $gray-400; }
+:root {
+  --base00: #{$white};
+  --base01: #{$gray-700};
+  --base02: #{$red-500};
+  --base03: #{$blue-600};
+  --base04: #{$green-700};
+  --base05: #{$cyan-800};
+  --base06: #{$gray-800};
+  --base07: #{$pink-200};
+  --base08: #{$green-900};
+  --base09: #{$green-200};
+  --base0A: #{$blue-900};
+  --base0B: #{$cyan-700};
+  --base0C: #{$orange-700};
+  --base0D: #{$purple-700};
+  --base0E: #{$teal-700};
+  --base0F: #{$black};
+  --base10: #{$gray-400};
+  --base11: #{$yellow-800};
+  --base12: #{$cyan-900};
+}
 
-.chroma {
-  background-color: $white;
+.hll { background-color: #fff; }
```

Comment:
Question asked to Bootstrap. I suppose it is `.hl` but I'm not sure. The white color is weird if that's the case :shrug: Should rather be yellow-ish.

Reply:
Modification done in https://github.com/twbs/bootstrap/pull/36173.